lib/versioning/loose/generic.ts

```diff
@@ -133,7 +134,51 @@ export abstract class GenericVersioningApi<
       : null;
   }
 
-  protected abstract _compare(version: string, other: string): number;
+  protected _compare(version: string, other: string): number {
```

Comment:
Any potential that this causes other versionings to now change behavior?

Reply:
no, only regex versioning is using this base class